### ISSUE TITLE
Performance data on health centers

### ISSUE THREAD

**Issue by cornstein** [demand-driven-open-data/ddod-intake#23]
HRSA tracks performance data on health centers it funds (http://bphc.hrsa.gov/policiesregulations/performancemeasures/). The data to accompany it should be made available online.

**Comment by dportnoy** [comment]
Created page for full use case specifications and solution: http://hhs.ddod.us/wiki/Use_Case_23

**Comment by Jordanrau** [comment]
This health data would be very valuable to consumers and insurers. I agree that it should be released and updated in a timely basis.

**Comment by betshsu** [comment]
@cornstein @Jordanrau Thanks for entering this use case; sorry for the delay in response. 

To help us structure this as a use case, could you describe how the data will be used and elaborate on the value it delivers (beyond simply making the data public)?

Thanks.

**Comment by dportnoy** [comment]
@cornstein @Jordanrau  As we work with the data owners on identifying whether and how this data could be released, it would be helpful if you could provide more context around it.  Doing so increases the likelihood and speed of fulfilling a request.  We're looking for specifics about how the data would be used, fields needed, and the value to yourself and public health.  Thanks!

**Comment by cornstein** [comment]
Hi there, it appears from the website included earlier that HRSA collects a wealth of data from health centers. I've pasted below the fields HRSA says it collects. This will allow media and the public to compare health center performance, look at oversight of centers by HRSA, identify those who practice at high levels and those in need of improvements and provide consumers who rely on the health centers with timely, accurate information on their quality. The fields include:

Quality of Care Measures
Percentage of prenatal care patients who entered treatment during their first trimester

Numerator: Number of women entering prenatal care at the health center or with the referral provider during their first trimester

Denominator (Universe): Total number of women seen for prenatal care during the year

Percentage of children with their 3rd birthday during the measurement year who are fully immunized before their 3rd birthday

Numerator: Number of children among those included in the denominator who were fully immunized before their 3rd birthday; a child is fully immunized if s/he has been vaccinated or there is documented evidence of contraindication for the vaccine or a history of illness for ALL of the following: 4 DTP/DTaP, 3 IPV, 1 MMR, 3 Hib, 3 HepB, 1VZV (Varicella), and 4 Pneumococcal conjugate, prior to her/his third birthday

Denominator: Number of all children with at least one medical visit during the reporting period, who had their 3rd birthday during the reporting period, or a sample of 70 of these children

Percentage of women 21–64 years of age who received one or more Pap tests to screen for cervical cancer

Numerator: Number of female patients 24–64 years of age receiving one or more documented Pap tests during the measurement year or during the 2 calendar years prior to the measurement year among those women included in the denominator; OR, for women who were 30 years of age or older at the time of the test who choose to also have an HPV test performed simultaneously, during the measurement year or during the 4 calendar years prior to the measurement year

Denominator: Number of all female patients age 24–64 years of age during the measurement year who had at least one medical visit during the reporting year, or a sample of these women

Percentage of patients aged 2 until 17 who had evidence of BMI percentile documentation AND who had documentation of counseling for nutrition AND who had documentation of counseling for physical activity during the measurement year

Numerator: Number of patients in the denominator who had their BMI percentile (not just BMI or height and weight) documented during the measurement year AND who had documentation of counseling for nutrition AND who had documentation of counseling for physical activity during the measurement year

Denominator: Number of patients who were one year after their second birthday (i.e., were 3 years of age) through adolescents who were aged up to one year past their 16th birthday (i.e., up until they were 17) at some point during the measurement year, who had at least one medical visit during the reporting year, and were seen by the health center for the first time prior to their 17th birthday OR a sample of these patients

Percentage of patients aged 18 and older who had documentation of a calculated BMI during the most recent visit or within the six months prior to that visit and if the most recent BMI is outside parameters, a follow-up plan is documented

Numerator: Number of patients in the denominator who had their BMI (not just height and weight) documented during their most recent visit OR within six months of the most recent visit AND if the most recent BMI is outside parameters, a follow-up plan is documented

Denominator: Number of patients who were 18 years of age or older during the measurement year, who had at least one medical visit during the reporting year, OR a sample of these patients

Percentage of patients age 18 years and older who were screened for tobacco use at least once during the measurement year or prior year AND who received cessation counseling intervention and/or pharmacotherapy if identified as a tobacco user

Numerator: Number of patients in the denominator for whom documentation demonstrates that patients were queried about their tobacco use one or more times during their most recent visit OR within 24 months of the most recent visit and received tobacco cessation counseling intervention and/or pharmacotherapy if identified as a tobacco user

Denominator: Number of patients who were 18 years of age or older during the measurement year, seen after 18th birthday, with at least one medical visit during the reporting year, and with at least two medical visits ever, OR a sample of these patients

Percentage of patients aged 5 through 40 with a diagnosis of mild, moderate, or severe persistent asthma who received or were prescribed accepted pharmacologic therapy

Numerator: Number of patients in the denominator who received a prescription for or were provided inhaled corticosteroid or an accepted alternative medication

Denominator: Number of patients who were 5 through 40 years of age at some point during the measurement year, who have been seen at least twice in the practice and who had at least one medical visit during the reporting year, who had an active diagnosis of persistent asthma OR a sample of these patients

Percentage of patients aged 18 years and older with a diagnosis of Coronary Artery Disease (CAD) who were prescribed a lipid-lowering therapy

Numerator: Number of patients in the denominator who received a prescription for or were provided or were taking lipid lowering medications

Denominator: Number of patients who were seen during the measurement year after their 18th birthday, who had at least one medical visit during the reporting year, with at least two medical visits ever, and who had an active diagnosis of coronary artery disease (CAD) including any diagnosis for myocardial infarction (MI) or who had had cardiac surgery in the past – OR a sample of these patients

Percentage of patients aged 18 years and older who were discharged alive for acute myocardial infarction (AMI) or coronary artery bypass graft (CABG) or percutaneous transluminal coronary angioplasty (PTCA) in the prior year OR who had a diagnosis of ischemic vascular disease during the measurement year who had documentation of use of aspirin or another antithrombotic

Numerator: Number of patients in the denominator who had documentation of aspirin or another anti-thrombotic medication being prescribed, dispensed or used

Denominator: Number of patients who were aged 18 and older at some point during the measurement year, who had at least one medical visit during the reporting year, who had an active diagnosis of ischemic vascular disease (IVD) during the current or prior year OR had been discharged after AMI or CABG or PTCA in the prior year—OR a sample of these patients

Percentage of patients aged 50 to 75 who had appropriate screening for colorectal cancer

Numerator: Number of patients aged 51 through 74 with appropriate screening for colorectal cancer

Denominator: Number of patients who were aged 51 through 74 at some point during the measurement year, who had at least one medical visit during the reporting year

Percentage of patients aged 12 and older screened for clinical depression using an age appropriate standardized tool AND follow-up plan documented

Numerator: Number of patients age 12 years and older who were (1) screened for depression with a standardized tool during the measurement year and, if positive, (2) had a follow-up plan documented

Denominator: Number of patients age 12 years and older who had at least one medical visit during the measurement year

Percentage of newly diagnosed HIV patients who had a medical visit for HIV care within 90 days of first-ever HIV diagnosis

Numerator: Number of patients in the denominator who had a medical visit for HIV care within 90 days of first-ever HIV diagnosis

Denominator: Number of patients first diagnosed with HIV between October 1 of the prior year through September 30, of the current measurement year

Percentage of children, age 6–9 years, at moderate to high risk for caries who received a sealant on a first permanent molar during the reporting period

Numerator: Subset of children in the denominator who received a sealant on a permanent first molar tooth in the measurement year

Denominator: Number of health center patients, age 6–9 years old, who had an oral assessment or comprehensive or periodic oral evaluation visit and are at moderate to high risk for caries in the measurement year

Health Outcomes and Disparities
Percentage of adult patients 18 to 75 years of age with a diagnosis of Type I or Type II diabetes, whose hemoglobin A1c (HbA1c) was greater than 9% at the time of the last reading in the measurement year or an HbA1c test was not done

Numerator: Number of adult patients whose most recent hemoglobin A1c level during the measurement year is > 9% among those patients included in the denominator

Denominator: Number of adult patients aged 18 to 75 as of December 31 of the measurement year with a diagnosis of Type I or II diabetes AND, who have been seen in the clinic for medical visits at least twice during the reporting year AND, do not meet any of the exclusion criteria OR a statistically valid sample of 70 of these patients

Percentage of patients 18 to 85 years of age with diagnosed hypertension (HTN) whose blood pressure (BP) was less than 140/90 at the time of the last reading

Numerator: Number of patients in the denominator whose last systolic blood pressure measurement was less than 140 mm Hg and whose diastolic blood pressure was less than 90 mm Hg

Denominator: All patients 18 to 85 years of age as of December 31 of the measurement year with a diagnosis of hypertension (HTN) AND who were first diagnosed by the health center as hypertensive at some point before June 30 of the measurement year AND who have been seen for medical visits at least twice during the reporting year OR a statistically valid sample of 70 of these patients

Percentage of patients born to health center patients whose birthweight was below normal (less than 2500 grams)

Numerator: Number of children born with a birthweight of under 2500 grams

Denominator: Number of children born

Financial Viability / Costs
Total cost per patient

Numerator: Total accrued cost before donations and after allocation of overhead

Denominator: Total number of patients

UDS Lines: T8AL17CC/T4L6A for existing grantees

Medical cost per medical visit

Numerator: Total accrued medical staff and medical other cost after allocation of overhead (excludes lab and x-ray cost)

Denominator: Non-nursing medical encounters (excludes nursing (RN) and psychiatrist encounters)

UDS Lines: (T8AL1CC + T8AL3CC)/(T5L15CB – T5L11CB) for existing grantees

Health Center Program grant cost per patient

Numerator: Total accrued BPHC section 330 grants drawn-down for the period from January 1 to December 31, of the calendar measurement year

Denominator: Total unduplicated patients for the period from January 1 to December 31, of the calendar measurement year

UDS Lines: T9EL1gCa/(T3AL39Ca + T3AL39Cb) for existing grantees

Change in net assets to expense ratio

Numerator: Ending Net Assets – Beginning Net Assets

Denominator: Total Expense

Note: Net Assets = Total Assets – Total Liabilities

Working capital to monthly expense ratio

Numerator: Current Assets – Current Liabilities

Denominator: Total Expense / Number of Months in Audit

Long term debt to equity ratio

Numerator: Long Term Liabilities

Denominator: Net Assets

**Comment by betshsu** [comment]
Thanks; I'll work on documenting this and start conversations with HRSA.

**Comment by betshsu** [comment]
Is the performance data of interest different than what is available here:

http://bphc.hrsa.gov/uds/datacenter.aspx

Full national report: http://bphc.hrsa.gov/uds/datacenter.aspx?q=tall&year=2014&state=
(Excel export available in upper right hand corner, though I have issues exporting the full report, but can export table by table)

State level data (with drill down to health center level): http://bphc.hrsa.gov/uds/datacenter.aspx?q=d

?

**Comment by cornstein** [comment]
There does not seem to be a way to access all the data for all health centers in a single spreadsheet. Is there? Am I missing it?

**Comment by marks** [comment]
@cornstein I am looking for the same thing... havent found it yet. Also, the export to excel seems to just put the HTML into a .xls file and hope that Excel can use its logic to extract data from the HTML `<table>` elements

**Comment by betshsu** [comment]
@cornstein No, I don't think there is a way to get all the health centers in a single spreadsheet.  Would you want all the info available for a single health center profile (e.g. http://bphc.hrsa.gov/uds/datacenter.aspx?q=d&bid=1011770&state=AK&year=2014) for all health centers in a single spreadsheet?  (though clearly, Excel is a suboptimal way to achieve this).  

I'll start talking to HRSA about other possible ways of making this data available, but the more specificity we can provide the better.

**Comment by marks** [comment]
Even just getting all the `bid`s and their state code would at least allow someone to scrape it. The XLS doesnt provide anything more structured or machine-readable than the HTML unfortunately

**Comment by cornstein** [comment]
@betshsu Yes, to be able to get all the elements from every profile into a single/multiple spreadsheets (doesn't have to be Excel) is exactly what I'm asking. Its clearly available in an electronic format and the site is clearly being powered by something like that, but it's not available easily to the public.

@marks You are right. The export tool is pretty worthless.

**Comment by betshsu** [comment]
@marks Thanks -- yeah, I'll bring up the format issue as well.  Perhaps they'll allow us to work on something internally if they don't have capacity to improve it themselves.

@cornstein Thanks.

**Comment by betshsu** [comment]
Updated the wiki with the information above -- feel free to edit -- and reached out to HRSA for a contact.  Will update as things progress.

**Comment by betshsu** [comment]
@cornstein I apologize for the lack of updates -- I haven't managed to get a response from the program owners to any of my inquiries, so I am attempting some other channels.

**Comment by betshsu** [comment]
Update -- we have a call scheduled with the program owner on Dec 8 to see whether and how the issues discussed here can be addressed

**Comment by betshsu** [comment]
@cornstein @marks 
Looks like HRSA took the complaints from multiple stakeholders about the health center data not being truly machine readable to heart and is working on improvements to the export of the health center data. The 2014 data is currently up; they are still working on exports of the national and state level aggregate data, but you can get machine readable data on the individual health centers by state in a single export -- after you choose a state, the first box will have a link to "download [state] aggregated health center data."  This will download an XLSX file with the following tabs (corresponding to the tables on the website):
- Age and Race-Ethnicity
- Patient Characteristics
- Services
- Clinical Data
- Cost
- Resource Information

Each center is a row in each table.
The overall state and national data (which was what was formerly exported as an HMTL dump into an Excel file) is currently not yet available as a machine-readable export, but HRSA is working on making those machine-readable exports and in the mean time have removed the "export to Excel" option on those pages.  This will hopefully be released in January, but the release date is not guaranteed.

In addition, HRSA is working on:
- Adding more back years to the trend information presented in the web tables (currently shows 3 year trends)
- Export of graphs in addition to tables

Some additional notes about the data:
- Data is typically updated and released to the public at the end of August 
- 2007 will likely remain the anchor year, as the oldest year for which data can be exported

I think this covers most of the issues that were brought up -- please let me know if something was missed.

**Comment by cornstein** [comment]
Hi Elizabeth. This is great. Is there any way to pull down the entire
nation in a single spreadsheet for those of us who do national
reporting/analysis?

Thanks,
Charlie

**Comment by betshsu** [comment]
@cornstein To clarify, when you reference the "entire nation in a single spreadsheet" would you just want an exportable form of the aggregate national tables on the website (e.g., the tables here: http://bphc.hrsa.gov/uds/datacenter.aspx?q=tall&year=2014&state=&fd=) or would you want the health center level data in the state reports (one row per health center) for the entire nation in a single export?

If it's the former, the HRSA contractor is currently working on that.  If it's the latter, we can inquire with the program owners to check on feasibility and see if we can get it added to the queue of improvements.

**Comment by cornstein** [comment]
It's the latter. Thanks. 

—

**Comment by betshsu** [comment]
Thanks for the confirmation -- we'll contact the program owners and see what can be done.

**Comment by betshsu** [comment]
@cornstein Currently, if a download for all US health center data is requested, the Bureau of Primary Health Care directs the requester to HRSA's FOIA office, after which a file with the data will be mailed.  

That said, the HRSA program owners did think it was worth exploring making this process available through the public-facing website -- we'll continue to investigate this option with them and post with updates, but if you'd like all the US data in a single file sooner rather than later, you might go with the FOIA option for now.

**Comment by cornstein** [comment]
Thanks. Given that you can download each state, it would make sense to make the whole country downloadable too. It would make sense for HRSA to model itself after CMS in this way.  Thanks. 

—
